libstorage/ZdbStorage.cpp

```diff
@@ -52,30 +50,32 @@ Entries::Ptr ZdbStorage::select(h256 _hash, int64_t _num, TableInfo::Ptr _tableI
     }
 
     Entries::Ptr entries = std::make_shared<Entries>();
-    auto it = valueList.begin();
-    for (; it != valueList.end(); ++it)
+    auto it = values.begin();
+    for (; it != values.end(); ++it)
     {
         Entry::Ptr entry = std::make_shared<Entry>();
-        for (size_t j = 0; j < it->size(); ++j)
+        auto it2 = it->begin();
+        for (; it2 != it->end(); ++it2)
         {
-            if (columns[j] == ID_FIELD)
+            if (it2->first == ID_FIELD)
```

Comment:
https://github.com/FISCO-BCOS/FISCO-BCOS/blob/623ba91f99ae7fab08971618378a74094dbd634d/libstorage/RocksDBStorage.cpp#L79-L85

can we do same as RocksDBStorage ?